csrankings-t.csv

```diff
@@ -524,6 +524,7 @@ Tianhong Dai,University of Aberdeen,https://www.abdn.ac.uk/people/tianhong.dai,9
 Tianjia Shao,Zhejiang University,http://tianjiashao.com,NOSCHOLARPAGE
 Tianlong Chen,University of North Carolina,https://tianlong-chen.github.io/,LE3ctn0AAAAJ
 Tianming Liu,University of Georgia,http://cobweb.cs.uga.edu/~tliu,92RPXm0AAAAJ
+Tianmin Shu,Johns Hopkins University,https://www.tshu.io/,YT_ffdwAAAAJ

```

Comment:
<img width="1239" alt="Screen Shot 2024-08-05 at 10 05 32 AM" src="https://github.com/user-attachments/assets/8406ce82-7814-41fa-afc1-f38286beccf2">

From this page: https://www.cs.jhu.edu/faculty/